### ISSUE TITLE
Long-winded

### ISSUE THREAD

**Issue by gvwilson** [swcarpentry/modern-scientific-authoring#18]
From @ctb:

```
Fascinating stuff, but super long-winded.  This is a better blog post than
a lesson, IMO.  Most of the stuff on how we got to where we are is not of
interest to most learners, except perhaps tangentially.

A lesson might show some pictures / demos of how same content can be
displayed in different ways, as a way of illustrating things like latex
and md.
```

**Comment by ctb** [comment]
(I was referring specifically to: https://swcarpentry.github.io/modern-scientific-authoring/01-mess.html)

**Comment by sprossiter** [comment]
I think this kind of 'interesting historical context/road' approach is always going to appeal heavily to some and confuse/distract others. I like it, but I can see ctb's point. Maybe a safer course is to split off the history into a separate page (or 'breakout' section at the bottom)?

**Comment by plantarum** [comment]
@sprossiter up to a point. But do we really need to start at cave painting and moveable type as an introduction to LaTeX? That's quite far removed from the topic.

**Comment by sprossiter** [comment]
@plantarum I agree that _many_ will find it offputting. (Maybe I came across as implying 'only a few'.)  Hence the idea of having it as a separate page or 'breakout' section at the bottom (i.e., have a straight-to-the-point intro. but then with some interesting historical background as an addendum). I still like it though!

**Comment by jooolia** [comment]
I agree with @ctb 's comments. While it was interesting to start with a historical perspective, if I were looking to learn about modern ways to produce a scientific manuscript I might skim over this first section. 

I found the callout in lines 63-67 a bit hard to follow. I had to read it 4 times and still not sure that I understood the message.

**Comment by npch** [comment]
+1 to @ctb and @sprossiter. The background history is interesting if this was the preface to a book, but as the lead-in to a lesson it feels like filler (there's nothing the student can do to change this, and in some sense it doesn't matter if they don't learn the historical context as long as they learn the current reasons why doing things in one way is better than another).

I would jump straight into the current state of affairs (I'd be tempted to start at "most people simply want to write..."), link to boxout of explanations and examples of drawing/typesetting/markup languages, and a link/boxout to "How did we end up here?" after this introduction to the current landscape.
